### PR TITLE
updated gitrepositories.md (fixes #341)

### LINKED ISSUE #341: [feature] use gui section for gitrepositories.md (TortoiseGit, etc.)
### Problem
Learning how to use git commands in the command line can be overwhelming for some. Some people may have experience in languages but not in git command line usage or GitHub in general. 
Using the command line can also slow down developer workflow or discourage otherwise would be interns from contributing. The time it takes to look up a fork reset command for instance. I would never memorize that unless it's simply 2 words. But sometimes it can be like 4 commands in a row. Also, I have to right-click to paste inside the bash/cmd prompt which takes additional time. When I can control-v into a gui instead. 

### Steps to reproduce the problem
N/A

### Screenshots
https://i.imgur.com/fvqp700.png

### Proposed solution
I don't remember off the top of my head if we can throw this in elsewhere. But we can throw it in somewhere about how to use a gui application like TortoiseGit to do the same job as outlined in git bash code. 

EDIT: Virtual Interns should definitely know what the commands are and learn them by using them. But once they know and have done it all. They might want to switch to a gui after that as shown above.  
EDIT2: Or maybe instead of a section. Maybe just an option for it mid-article with a link to a guide on how to use a gui program instead. And not Github's gui because it lacks the ability to revert commits (and hard-reset). But TortoiseGit which can do everything the bash can do and more with right-click.

### PR DESCRIPTION
**Pull request by LordJashin32**
### Description
Added link to Git GUI's, a Github fork 1 click sychronizer, and
added the pretense of "reward" for the long steps climbed so far for 
new aspiring VI's. 

### Raw.Githack preview link
https://raw.githack.com/LordJashin32/lordjashin32.github.io/tristans-best-branch-1/#!pages/vi/gitrepositories.md

### CODE REVIEW

**Review by kjong — APPROVED** [review]

**Comment by irisb1701** [comment]
It's a nice addition, but probably deserves a tutorial under the blog section rather than get stashed at the bottom of step 5. I would maybe just add the git gui link underneath the useful links section for now, what do you think @dogi

**Comment by LordJashin32** [comment]
I'm going to have to change my mind on this a bit. It seems GitExtensions multi-platform is really bad. I think I'm going to just have to list what works best for each platform in particular. 
This Rabbit thing works just like tortoisegit: https://i.imgur.com/Re4Kdim.png .

**Comment by LordJashin32** [comment]
> 
> 
> It's a nice addition, but probably deserves a tutorial under the blog section rather than get stashed at the bottom of step 5. I would maybe just add the git gui link underneath the useful links section for now, what do you think @dogi

I think you're right it would be better to have a bigger section to explain the overarching importance of this in developer workflow. I'll leave the git gui links though because that's more general and not too specific like I was doing recommending specific programs. 
EDIT: Here's the final: https://raw.githack.com/LordJashin32/lordjashin32.github.io/1b36c451ab476e1a540c26565199fea6491266b8/#!pages/vi/gitrepositories.md

File: pages/vi/gitrepositories.md [hunk 1]
```diff
@@ -218,6 +218,7 @@ If you would like to understand how syncing with the fork works, here is a usefu
 [Git-it Workshop](http://jlord.us/git-it/) - Runs in your terminal to work and provides a hands-on approach to learn Git and GitHub repositories.
 [Git help](https://git-scm.com/) - An encyclopedia of useful git workflows and terminology explanations.
 [Other helpful links and videos](faq.md#Helpful_Links)
+[List of Git Graphical User Interfaces](https://git-scm.com/downloads/guis)
```
Comment (irisb1701) [inline pages/vi/gitrepositories.md]: these two just need to be reversed

**Review by irisb1701 — APPROVED** [review]

**Review by phamduchongan93 — APPROVED** [review]

**Comment by LordJashin32** [comment]
Sorry, this PR took a while. Should rename this to tristan's worst branch. I've been sick like 2-3 days out of the past week it just won't stop raining here. 

I like these changes and I hope new VIs will see this link and use the GUIs available after they are done learning the command line. Or even while they learn it. It helped me a great amount. Multiple ways to do the same thing is good.

**Review by dogi — APPROVED** [review]

**Pull request merged**